Teamcity build

sambhav-jain-16 opened the pull request.

sambhav-jain-16 committed:
roachtest: add code changes to benchmarks to emit openmetrics

https://github.com/cockroachdb/cockroach/pull/129221 and
https://github.com/cockroachdb/cockroach/pull/132023 added changes to
exporters to emit openmetrics. This PR makes changes to the roachtests
to make use of the changes in the above PRs. This change also made some
changes to some roachtests that use neither of the above approaches

Epic: https://cockroachlabs.atlassian.net/browse/CRDB-41852

Release note: None

sambhav-jain-16 committed:
roachtest: add code changes to benchmarks to emit openmetrics

https://github.com/cockroachdb/cockroach/pull/129221 and
https://github.com/cockroachdb/cockroach/pull/132023 added changes to
exporters to emit openmetrics. This PR makes changes to the roachtests
to make use of the changes in the above PRs. This change also made some
changes to some roachtests that use neither of the above approaches

Epic: https://cockroachlabs.atlassian.net/browse/CRDB-41852

Release note: None

sambhav-jain-16 committed:
build: add teamcity job

<what was there before: Previously, ...>
<why it needed to change: This was inadequate because ...>
<what you did about it: To address this, this patch ...>

(cherry picked from commit cfc5bb0c2312b71fe7998ae9ce32c84c6c0b7cae)
(cherry picked from commit 48209233301a1eb7880427b974dc9a28f742b9bd)

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/134621)
<!-- Reviewable:end -->

sambhav-jain-16 committed:
roachtest: use defer for exporter close in roachtests

Epic: none

Release note: None

(cherry picked from commit f44356d16701c47a92a9f200aeae8ebd6988185c)

sambhav-jain-16 committed:
clusterstats: rename output file name in openmetrics

Currently, the file that gets emitted by `clusterstats.Export()` is
`openmetrics.om`. To make it consistent with other roachtests, it should
be renamed to `stats.om`. This change intends to do it.

Epic: none

Release note: None

sambhav-jain-16 committed:
roachtest: add openmetrics emission to sysbench

This change adds capability of emitting openmetrics from sysbench benchmark

Epic: none

Release note: None
